addons/onboarding/tests/__init__.py

```diff
@@ -1,5 +1,6 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
+from . import case
```

Comment:
Note that this kind of content is generally put in "common"

Reply:
"We generally don't like to have multiple classes in the same file (even when it is done in other modules ^^)" who is this we :D ?

As you prefer anyway, not really an important matter.